### DISCUSSION TITLE
Sulu vhost for CloudPanel

### DISCUSSION THREAD

**Discussion by matthieu2607** [sulu/sulu#6823] [Ideas]
Hi,

I have fork a vhost repository for CloudPlanel host to add a sulu vhost.

Cloudpan runs with nginx.

The configuration that I will propose works on a real project (see the code : https://github.com/Pixel-Developpement/vhost-templates/commit/5c867173aa362e8315b58675bb1c40efbf542b35).

Do you see any improvements to be made before submitting this?

Thank you in advance for your response,

Best regards,

Matthieu

**Comment by alexander-schranz** [comment]
The official nginx vhost you can find here: https://docs.sulu.io/en/latest/cookbook/web-server/nginx.html which we recommend for your usecase.
